test/v5/kyberBancorReserve.js

```diff
@@ -123,56 +150,79 @@ contract('KyberBancorNetwork', function(accounts) {
         let result;
 
         result = await reserve.getConversionRate(ethAddress, bancorBntToken.address, amountETH, 0);
-        Helper.assertEqual(result, ethToBntRate, "should return correct eth to bnt rate");
+        Helper.assertEqual(result, Helper.addBps(ethToBntRate, -feeBps), "should return correct eth to bnt rate");
 
         result = await reserve.getConversionRate(bancorBntToken.address, ethAddress, amountBNT, 0);
-        Helper.assertEqual(result, bntToEthRate, "should return correct bnt to eth rate");
+        Helper.assertEqual(result, Helper.addBps(bntToEthRate, -feeBps), "should return correct bnt to eth rate");
```

Comment:
how did it pass before without reducing the fees?
it was only 0?

Reply:
I guess we can actually move the fee all together by now.
save some gas for users.
and reduce tests.

Reply:
sure, I think we can remove fee bps right?